walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/VerifiedIdPresentationRequest.kt

```diff
@@ -16,4 +16,6 @@ interface VerifiedIdPresentationRequest : VerifiedIdRequest<Unit> {
 
     // Completes the request and returns nothing if successful.
     override suspend fun complete(): VerifiedIdResult<Unit>
+
+    fun getNonce(): String
```

Comment:
I don't think this is required; not all request formats have nonces (although they should) so at the least having this return `String?`.

Reply:
I believe they should. But made it a nullable field.